layouts/partials/tw/topper.html

```diff
@@ -73,12 +73,12 @@
     {{ end }}
     {{ with $link }}
       <a
-        class="is-hidden-tablet"
+        class="sm:hidden"
         data-ga-category="mobile"
         href="{{ . }}"
       >
     {{ else }}
-      <span class="is-hidden-tablet">
+      <span class="sm:hidden">
```

Comment:
There is already tw/promo-topper.html, so you can just use that and not create a new file.

Reply:
👍 I also renamed the topper-article to match the naming convention